lenguasdearagon/aragonario-frontend
# Feature/conjugator options

# Description
Add options to conjugator.
# How to test
Go to ```/words/<verb>/conjugation```, test different options and check that conjugations table changes according to selected options.
